juniper_official/Solutions/EVPN-VXLAN/Vcenter/check-vcenter-host-mac-mapping.rule

```diff
@@ -3,18 +3,34 @@
  * by querying database of rule vcenter.evpn/get-vcenter-details and displays entire
  * path in dashboard.
  *
- * Four input controls detection
- * 
- *   1) db-server-ip, user need to enter HB DB server ip while deploying 
- *      playbook.
- *   2) db-port, user need to enter HB DB server udp port while deploying
- *      playbook.
- *   3) vcenter-device-group, user need to enter vcenter device group name 
- *      i.e. where playbook "get-vsphere-details" deployed.
- *   4) vcenter-device, user need to enter vcenter device name which should
- *      be part of device group.
+ * This rule to work, users need to create tagging profile and apply to device group.
+ * Tagging profile will create fields "vcenter-device" and "vcenter-device-group".
  *
- */
+ * Below is a sample tagging profile
```

Comment:
For now, can you also add the dummy non-key field in the example (given https://ssd-git.juniper.net/iceberg/healthbot-core/-/issues/463)?